File: rest/test-credentials/test-sms-messages-example-2/test-sms-messages-example-2.rb
# Get twilio-ruby from twilio.com/docs/ruby/install
require 'rubygems'          # This line not needed for ruby > 1.8
require 'twilio-ruby'

# Get your Account Sid and Auth Token from twilio.com/user/account
account_sid = '{{ account_sid }}'
auth_token = '{{ auth_token }}'
@client = Twilio::REST::Client.new account_sid, auth_token

sms = @client.account.sms.messages.create(:body => "Hey Mr Nugget, you the bomb!",
    :to => "+15005550009",
    :from => "+15005550006")
puts sms.from

File: rest/test-credentials/test-sms-messages-example-2/test-sms-messages-example-2.py
# Download the Python helper library from twilio.com/docs/python/install
from twilio.rest import TwilioRestClient

# Your Account Sid and Auth Token from twilio.com/user/account
account_sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
auth_token  = "{{ auth_token }}"
client = TwilioRestClient(account_sid, auth_token)

sms = client.sms.messages.create(body="Hey Mr Nugget, you the bomb!",
    to="+15005550009",
    from_="+15005550006")
print sms.sid
